addons/pandora/backend/entity_backend.gd

```diff
@@ -215,11 +249,17 @@ func _deserialize_entities(data:Array) -> Dictionary:
 		# otherwise rely on the script path of the parent category.
 		if entity_data.has("_script_path"):
 			var entity = _create_entity_from_script(entity_data["_script_path"], "", "", "", "")
+			if entity == null:
```

Comment:
again, in case there was any problem with creating an entity object, do treat it as an error.